src/v/cloud_storage/partition_manifest.cc

```diff
@@ -599,10 +760,16 @@ struct partition_manifest_handler
         expect_segment_meta_start,
         expect_segment_meta_key,
         expect_segment_meta_value,
+        expect_replaced_start,
+        expect_replaced_path,
+        expect_replaced_meta_start,
+        expect_replaced_meta_key,
+        expect_replaced_meta_value,
         terminal_state,
     } _state{state::expect_manifest_start};
 
     using segment_map = partition_manifest::segment_map;
+    using segment_multimap = partition_manifest::segment_multimap;
```

Comment:
it looks like this compile won't compile because `partition_manifest::segment_multimap;` isn't defined until a later commit.